.bazelrc

```diff
@@ -145,4 +145,5 @@ build:remote-ci --remote_executor=grpcs://remotebuildexecution.googleapis.com
 build:asan-fuzzer --config=asan
 build:asan-fuzzer --define=FUZZING_ENGINE=libfuzzer
 build:asan-fuzzer --copt=-DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION
-build:asan-fuzzer --copt=-fsanitize-coverage=trace-pc-guard
\ No newline at end of file
+build:asan-fuzzer --copt=-fsanitize-coverage=trace-pc-guard
+build:asan-fuzzer --test_env=UBSAN_OPTIONS=print_stacktrace=1
```

Comment:
Add a comment why this is overridden and perhaps link to protobuf issue (is that a real issue in protobuf?)

Reply:
Hm, I haven't seen anyone make an issue about this, although some of the protobuf code isn't tested with UBSan, it might need a fix upstream there + patch until release.